src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/InputPane/InputPane_Simple.xaml.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Uno.UI.Samples.Controls;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+#if __ANDROID__
+using Android.App;
+using Android.Views;
+#endif
+
+// The User Control item template is documented at https://go.microsoft.com/fwlink/?LinkId=234236
+
+namespace UITests.Shared.Windows_UI_Xaml_Controls.InputPane
+{
+	[SampleControlInfo("InputPane", "InputPane_Simple")]
+	public sealed partial class InputPane_Simple : UserControl
+	{
+		private Windows.UI.ViewManagement.InputPane _inputPane;
+
+		public InputPane_Simple()
+		{
+			this.InitializeComponent();
+
+			_inputPane = Windows.UI.ViewManagement.InputPane.GetForCurrentView();
+			_inputPane.Showing += (s, e) => Update();
+			_inputPane.Hiding += (s, e) => Update();
+
+			Update();
+		}
+
+		private void Update()
+		{
+			IsVisible.Text = $"IsVisible: {_inputPane.Visible.ToString()}";
+			OccludedRect.Text = $"OccludedRect: {_inputPane.OccludedRect.ToString()}";
+		}
+
+		private void TryShow(object sender, RoutedEventArgs args) => _inputPane.TryShow();
+
+		private void TryHide(object sender, RoutedEventArgs args) => _inputPane.TryHide();
+
+		private void ShowNavigationBar(object sender, RoutedEventArgs args)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826544)